pipelines/smartseq2_multisample/README.md

```diff
@@ -1,12 +1,64 @@
-# SmartSeq2 scRNA-Seq Data processing Pipeline for Multiple Cells
-
-This pipeline performs the following tasks:
-- Runs the SS2 pipeline for each cell
-- Merges the resulting Zarr output files
- 
-# Inputs
- - Inputs are identical for the SS2 pipeline however, for running 
- multiple samples a URL prefix is provided along with an array of filename prefixes 
- under that path. File names are autogenerated by appending _1.fastq.gz and (in the case of 
- paired end) _2.fastq.gz at the end of the filenames
- - Note that single and paired end samples can not be mixed in the same run
\ No newline at end of file
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| v.1.0.0 | December, 2019 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in skylab or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+
+# Smart-seq2 Multi Sample Pipeline
+## Introduction
+The Smart-seq2 Multi Sample pipeline is modified from the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is designed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files. 
```

Comment:
is modified --> is a wrapper around the 

designed --> developed

I am not sure if this is an HCA pipeline, check with @kbergin 

then merging the resulting Zarr output files **and optionally generating a loom file**